src/schema/provenance_schema.yaml

```diff
@@ -594,6 +600,12 @@ ENTITIES:
         generated: true
         immutable: true
         description: "The assay types of this Dataset. Valid values are from UBKG are queried by schema_manager.get_valueset_dataset_type() using the Ontology API."
+      display_subtype:
```

Comment:
Publication is considered as a subclass of Dataset, since `display_subtype` is defined for Dataset, no need to redefine for Publication.